### PR TITLE
Logout network call

### PR DESCRIPTION
**Pull request by sjl1826**
Made logout network call

### CODE REVIEW

File: Bruin Bite/Network/UserManager.swift [hunk 1]
```diff
@@ -172,6 +172,19 @@ class UserManager {
         }
     }
 
+    func invalidateAccessToken(token: String) {
```
Comment (HirdayGupta) [inline Bruin Bite/Network/UserManager.swift]: rename this function to `logOutUser`and just copy the two lines from the current `logOutUser()` into this function.

File: Bruin Bite/Network/UserManager.swift [hunk 2]
```diff
@@ -172,6 +172,19 @@ class UserManager {
...
+        self.provider.request(.logoutUser(token: token, client_id: CLIENTID, client_secret: CLIENTSECRET)) { result in
+            switch result {
+            case .success:
+                self.logOutUser()
```
Comment (HirdayGupta) [inline Bruin Bite/Network/UserManager.swift]: Add the two lines from the old logOutUser function here after changing this functions name

**Review by HirdayGupta — CHANGES_REQUESTED** [review]
Great work, small change requested!

File: Bruin Bite/Network/UserManager.swift (hunk 1 above)
Comment (sjl1826) [reply Bruin Bite/Network/UserManager.swift]: Gotcha

File: Bruin Bite/Network/UserManager.swift (hunk 2 above)
Comment (sjl1826) [reply Bruin Bite/Network/UserManager.swift]: Okie

**Review by HirdayGupta — APPROVED** [review]
Looks good!

**Pull request merged**